fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVPartitionExprDateTrunc.java

```diff
@@ -198,9 +198,15 @@ private DateTimeV2Literal dateIncrement(DateTimeV2Literal value) throws Analysis
             case "year":
                 result = value.plusYears(1L);
                 break;
+            case "quarter":
+                result = value.plusMonths(3L);
```

Comment:
why add 3 month, can add some comment to explain this

Reply:
1 quarter = 3month

Reply:
value not have `plusQuarter` method,so use plusMonths(3L);